pkg/scheduler/framework/runtime/framework.go

```diff
@@ -346,12 +346,6 @@ func NewFramework(r Registry, profile *config.KubeSchedulerProfile, stopCh <-cha
 		}
 	}
 
-	if len(f.queueSortPlugins) == 0 {
-		return nil, fmt.Errorf("no queue sort plugin is enabled")
-	}
-	if len(f.queueSortPlugins) > 1 {
-		return nil, fmt.Errorf("only one queue sort plugin can be enabled")
-	}
```

Comment:
Thanks for your work. But this has already been removed here https://github.com/kubernetes/kubernetes/pull/110167/files#diff-0aa9024787a576d2bfcd5766f717a75b88978dbd1c927bf4139faa91b508ddddL349-L358.